docs/source/user_guide/memory.rst

```diff
@@ -198,3 +198,60 @@ Be sure to do this before any other CuPy operations.
 
    # Disable memory pool for pinned memory (CPU).
    cupy.cuda.set_pinned_memory_allocator(None)
+
+
+System allocated memory (SAM) support (**experimental!**)
```

Comment:
ditto, we might not want to call it SAM here

Reply:
changed to UMP in [846682f](https://github.com/cupy/cupy/pull/8442/commits/846682f1d23deaf109b3edbf7d49fb5f520d735e)